tests/foreman/cli/test_repository.py

```diff
@@ -560,6 +562,34 @@ def test_positive_create_docker_repo_with_name(self, repo_options, repo):
         for key in 'url', 'content-type', 'name':
             assert repo.get(key) == repo_options[key]
 
+    @pytest.mark.tier1
+    def test_positive_create_with_new_organization_and_location(self):
+        """Check if repo can be created with a new Organization and Location.
+
+        :parametrized: no
```

Comment:
@sambible you will also have to add `:id:`. You can fix it by running `pre-commit run -a` or 
```
`In [1]: import uuid

In [2]: print(uuid.uuid4())
04af6170-2eed-4d0d-be8d-b5b86c7ce678
```
Non blocking comment: You can add test steps(`:Steps:`) as well.